scripts/generate-openapi-bindings-bdd.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -e
+
+ROOT_DIR="$(dirname "$0")/.."
+echo $ROOT_DIR
+TARGET="$ROOT_DIR/tests/bdd/openapi"
+SPEC="$ROOT_DIR/control-plane/rest/openapi-specs/v0_api_spec.yaml"
+
+# Regenerate the bindings only if the rest src changed
```

Comment:
don't think we need this since we're not committing the files?

Reply:
Removed.